src/Symfony/Bundle/FrameworkBundle/Resources/config/rate_limiter.php

```diff
@@ -26,5 +27,7 @@
                 abstract_arg('storage'),
                 null,
             ])
+
+        ->alias(RateLimiterFactoryInterface::class, 'limiter')
```

Comment:
we have to make sure that we only add this alias when the interface exists (which is not the case when the RateLimiter is installed in an older version)

Reply:
Indeed 👍